sktime/datatypes/_utilities.py

```diff
@@ -77,7 +78,28 @@ def get_index_for_series(obj, cutoff=0):
     return pd.RangeIndex(cutoff, cutoff + obj.shape[0])
 
 
-def get_cutoff(obj, cutoff=0, return_index=False):
+GET_CUTOFF_SUPPORTED_MTYPES = [
```

Comment:
```suggestion
CUTOFF_SUPPORTED_MTYPES = [
```
This is not a function, just a constant list. Why `GET_`?

Reply:
this refers to the function `get_cutoff`, i.e., the list of mtypes that is supported by the function `get_cutoff`

Reply:
if I read it like that, ok, accepted